Automattic/jetpack
# Build: remove unneeded files from packages' production builds.

#### Changes proposed in this Pull Request:

* This should ensure that we do not ship unneeded files in production builds of some of our packages.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Check that the added files are files that are not needed in a production build.
